pkgs/by-name/cr/crow/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  runCommand,
```

Comment:
Remove this if this is not used

Reply:
ahh good catch i needed it before i figured out that `doCheck = true;` will do the trick